operator/api/v1alpha2/keptnapp_types.go

```diff
@@ -31,8 +31,8 @@ import (
 // KeptnAppSpec defines the desired state of KeptnApp
 type KeptnAppSpec struct {
 	Version string `json:"version"`
-	// +kubebuilder:default:="1"
-	Revision                  string             `json:"revision,omitempty"`
+	// +kubebuilder:default:=1
+	Revision                  int                `json:"revision,omitempty"`
```

Comment:
in this PR I don't see any code that changes this field. Did you forgot to refactor something? 🤔

Reply:
the code does not exists yet, this is a preparation for #544